src/platform_impl/macos/app_state.rs

```diff
@@ -16,18 +16,21 @@ use icrate::Foundation::{is_main_thread, NSSize};
 use objc2::rc::{autoreleasepool, Id};
 use once_cell::sync::Lazy;
 
-use super::appkit::{NSApp, NSApplication, NSApplicationActivationPolicy, NSEvent};
+use super::{
+    appkit::{NSApp, NSApplication, NSApplicationActivationPolicy, NSEvent},
+    event_loop::ControlFlow,
+};
 use super::{
     event_loop::PanicInfo, menu, observer::EventLoopWaker, util::Never, window::WinitWindow,
 };
 use crate::{
     dpi::PhysicalSize,
     event::{Event, InnerSizeWriter, StartCause, WindowEvent},
-    event_loop::{ControlFlow, EventLoopWindowTarget as RootWindowTarget},
+    event_loop::EventLoopWindowTarget as RootWindowTarget,
     window::WindowId,
 };
 
-static HANDLER: Lazy<Handler> = Lazy::new(Default::default);
+pub(crate) static HANDLER: Lazy<Handler> = Lazy::new(Default::default);
```

Comment:
Nit: So far we've only exposed this to other modules through `AppState`, I think I'd like to keep it that way (makes it easier to refactor later on).

Reply:
(So add a `set_control_flow` method to `AppState` instead)

Reply:
Done.